crates/ruff_python_ast/src/nodes.rs

```diff
@@ -1531,6 +1541,12 @@ impl StringLiteralFlags {
         self
     }
 
+    #[must_use]
+    pub fn with_invalid(mut self) -> Self {
```

Comment:
Could possibly call this method `marked_as_invalid`? (Same for the one on `BytesLiteralFlags`)